src/components/tuits/update-tuit.js

```diff
@@ -15,6 +15,13 @@ const TuitScreen = () => {
     return(
         <div>
 
+            <h4>Old Tuit:</h4>
+            <a>{tuit.tuit}</a>
+
+            <br/>
+            <br/>
+
```

Comment:
This functionality is working for me.  I agree with Katie C., let's add the alert idea.

Reply:
Alert idea implemented

Reply:
Woot!